### PR TITLE
Add xclbin.h into XoclUser Windows project

### PR DESCRIPTION
**Pull request by igorshr**
Added xclbin.h into the Windows XoclUser project. Point all possible refernces into the new file. Because XoclUser_INTF.h is changed, renaming some structures to use xclbin.h names, I changed ship.cpp also. All changes in ship.cpp are minor and not reflect functionality. However I tested changes by running "xbutil program -p <file to reprogram>".

### CODE REVIEW

**Comment by maxzhen** [comment]
ok to test

File: src/runtime_src/core/pcie/driver/windows/include/XoclUser_INTF.h [hunk 1]
```diff
@@ -234,27 +235,11 @@ typedef GUID xuid_t;
...
-    UCHAR m_tag[16]; //DDR: BANK0,1,2,3, has to be null terminated; if streaming then stream0, 1 etc
-
-} XU_MEM_TOPO_DATA, *PXU_MEM_TOPO_DATA;
-
 typedef struct _XOCL_MEM_TOPOLOGY_INFORMATION {
 
     ULONG        MemTopoCount;
 
-    XU_MEM_TOPO_DATA MemTopo[XOCL_MAX_DDR_BANKS];
+    struct mem_data MemTopo[XOCL_MAX_DDR_BANKS];
```
Comment (stsoe) [inline src/runtime_src/core/pcie/driver/windows/include/XoclUser_INTF.h]: Isn't this wrong?     There is no limit to the number of banks.  
Also, should this be per xclbin.h, e.g. struct mem_topology should be used from xclbin.

Comment (arpitxilinx) [reply src/runtime_src/core/pcie/driver/windows/include/XoclUser_INTF.h]: Yes, I agree this should be per xclbin data struct. I talked with Igor and he mentioned, this was an oversight and he will fix it.

Comment (igorshr) [reply src/runtime_src/core/pcie/driver/windows/include/XoclUser_INTF.h]: Fixed. I removed XOCL_MEM_TOPOLOGY_INFORMATION from XoclUser_INTF.h and made references to struct mem_topology of xclbin.h

**Comment by igorshr** [comment]
> Hello Igor,
> 
> While not blocking, could you help me understand why you are commenting out code instead of just removing it?
> 
> Thanks, Stephen

Fixed. Commented code removed.

**Pull request merged**